build/build-constants.yaml

```diff
@@ -85,6 +85,7 @@
 - name: file-paths
   constants:
     build-manifest-file-name: oryx-manifest.toml
+    run-script-generator-dir: /opt/startupcmdgen
```

Comment:
Change this PR's base branch to the branch you created for PR # 194...this way you can only see test changes in this PR

Reply:
Done